File: Day 7/453. Minimum Moves to Equal Array Elements.java
class Solution {
    public int minMoves(int[] nums) {
        int min = nums[0];
        for (int i =1;i <nums.length;i++){
            min = Math.min(min , nums[i]);
        }
        int ans =0;
        for (int i=0; i < nums.length;i++){
            ans += nums[i]-min;
        }
        return ans;
    }
}

File: Day 7/453. Minimum Moves to Equal Array Elements.cpp
class Solution {
public:
    int minMoves(vector<int>& nums) {
        int m =nums[0];
        for (int i =1;i <nums.size();i++){
            m = min(m , nums[i]);
        }
        int ans =0;
        for (int i=0; i < nums.size();i++){
            ans += nums[i]-m;
        }
        return ans;
    }
};